vars/main.yml

```diff
@@ -32,3 +32,22 @@ rhel7stig_re_qp_key_end: (?:" *)
 
 # insert the parameter at the beginning or append to the end, default append
 rhel7stig_re_qp_insert: "{{ insert | default(not (append | default(true))) }}"
+
+#Developers said the STIG lockdowns made ssh too difficult as the CAT2 finding says we must disconnect after 10 minutes of inactivity on ssh and console sessions. This will not be done on production/integration environments, but if teams want it in their sandbox, modify the following values:
+# rhel_07_040320_ssh_keepalive_interval:
+# rhel_07_040160_console_keepalive_interval:
+#Both values are in seconds (60 seconds * 10 = 600)
+             
+#The following will set the ssh timeout to 1 day:
+#RHEL-07-040320 | All network connections associated with SSH traffic must terminate at the end of the session or after 10 minutes of inactivity, except to fulfill documented and validated mission requirements
+rhel_07_040320_ssh_keepalive_interval: 28800
```

Comment:
This is a good add but should be placed in `defaults/main.yml` and set to the default setting that meets the STIG `600` and then overridden in your group or host vars files, not here in `vars/main.yml`.